### ISSUE TITLE
APNIC & RPKI publication service

### ISSUE THREAD

**Issue by magicnat** [NLnetLabs/rpki-doc#39]
We noticed that the Krill documentation states [the following](https://github.com/NLnetLabs/rpki-doc/blob/64a9b97ae7e14f8bcd19be3004f1505f23880a73/source/krill/before-you-start.rst#publishing-with-your-parent):

> At this time, only Asia Pacific RIR APNIC and Brazilian NIR NIC.br offer a publication server for their members. Several other RIRs have this functionality on their roadmap. This means that in most cases, you will have to publish yourself.

Since we are an APNIC member, we figure it worth a try. APNIC however, informed us that they do not offer such a service yet. Quoting George from APNIC:

> The publication server that we have is not compatible with Krill and actually for now it is only meant to be used by CNNIC, so it's not a "public service for members" yet. The information found in the Krill documentation is wrong and we will ask them (the Krill devs) to remove it to avoid this sort of confusion.
>
> So in summary we don't provide such service (yet) and never did.

**Comment by AlexanderBand** [comment]
Thanks for letting us know, I must have misunderstood earlier communication. I have updated the docs.

**Comment by magicnat** [comment]
It appears that APNIC started offering this; we migrated one of our repos to APNIC earlier, but their repo server implementation seems broken.

The exchange log says `APNIC::RPKI::Publication::CGI::post_message: DBIx::Class::Row::insert(): DBI Exception: DBD::mysql::st execute failed: Data too long for column 'request' at row 1` where the content of `request` field appears to be content of the ROAs. We have opened a ticket with APNIC regarding this.
